task/clamav-scan/0.1/clamav-scan.yaml

```diff
@@ -8,6 +8,8 @@ metadata:
     tekton.dev/tags: "virus, appstudio, hacbs"
   name: clamav-scan
 spec:
+  description: >-
+    Scan antivirus on the files extracted from the image
```

Comment:
```suggestion
    Scan the image for potential malware with clamav
```

Reply:
what about "Use ClamAV to scan the image for malware." ?